addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -93,7 +93,7 @@
 }
 
 // Force field label pointer to cursor
-.form-check, .form-select {
+.form-check, .form-select:not(:disabled) {
```

Comment:
Especially true in stable but in this case maybe also in master (not sure I made up my mind about that yet), I would not increase the specificity of rules that way. You can do:
```scss
.form-select:where(:not(:disabled)) {
```
It allows to target the same things but the specificity of the rule is not increasing by the "not" part. That way if someone made something like
```scss
.super-cursor {
    pointer: super;
}
```
and add it on a specific form-select, your fix here would not break his super cursor.